### ISSUE TITLE
dell_hw_nic_status metric returning 1 for functional interfaces

### ISSUE THREAD

**Issue by B0go** [galexrt/dellhw_exporter#94]
Hi @galexrt, thanks for this great contribution to the community. Such an exporter is essential for Dell based  on-prem observability! 

While analyzing the values returned by the exporter from a couple of different servers, I noticed that some metrics were returning 1s for network interfaces that were known to be functional. After digging into the code, I [found](https://github.com/galexrt/dellhw_exporter/blob/main/pkg/omreport/omreport.go#L454) that it only marks an interface as functional (0) when the report either returns "Connected" or "Full" in the fifth field. While this covers most scenarios, some edge cases are not covered.

Here is an example of a Team Interface that currently shows 1 in the metric but is functional:

```
Index             : 1
Interface Name    : br0
Vendor            : Linux
Description       : Network Bridge
Redundancy Status : Not Applicable
```

I believe this happens because "Not Applicable" is not covered by the exporter logic. It would be a good improvement to also check the field name before matching its value.

I can open a Pull Request including this scenario!

**Comment by galexrt** [comment]
@B0go A pull request to fix this would be appreciated. Please add a test for this case as well, thanks!

Let me know if you need any guidance.

**Comment by B0go** [comment]
@galexrt I will work on that whenever possible, thanks.

**Comment by galexrt** [comment]
I have just released `v1.13.2` containing your PR, see https://github.com/galexrt/dellhw_exporter/releases/tag/v1.13.2.

Please let me know if your fix works as expected.

**Comment by B0go** [comment]
Confirmed!
<img width="2543" alt="Screenshot 2024-02-07 at 10 15 36" src="https://github.com/galexrt/dellhw_exporter/assets/14275767/cc3775b1-af23-4099-963c-b8f01f797ac6">
